agent/config/config.go

```diff
@@ -80,6 +80,9 @@ const (
 	// minimumNumImagesToDeletePerCycle specifies the minimum number of images that to be deleted when
 	// performing image cleanup.
 	minimumNumImagesToDeletePerCycle = 1
+
+	// Default cgroup prefix for ECS tasks
+	DefaultTaskCgroupPrefix = "/ecs"
```

Comment:
This is the not going to be configurable now. 
The agent will create this prefix and stick all the tasks under here when the feature is enabled. 

The tasks that are started without this feature will be under the default cgroup root.